docs/pages/setup/reference/config.mdx

```diff
@@ -574,6 +574,10 @@ ssh_service:
       # display_offset + 1000 by default
       max_display: 1010
 
+    # Enables/disables remote file operations via SCP/SFTP for this node. Default
```

Comment:
```suggestion
    # Enables/disables remote file operations via SCP/SFTP for this Node. Default
```
We're capitalizing Node as a product name

Reply:
Thanks, done